plugins/janus_streaming.c

```diff
@@ -1096,6 +1096,7 @@ typedef struct janus_streaming_rtp_source {
 	janus_streaming_buffer *curldata;
 	char *rtsp_url;
 	char *rtsp_username, *rtsp_password;
+        char *rtsp_stream_uri;
```

Comment:
Code style: please use tabs and not spaces for indentation.

Reply:
Lots of little code style things -- I blame muscle memory. I'll fix these up, and add the config option, hopefully tonight.